### PR TITLE
Fixes for simple-processor example

### PR DESCRIPTION
**Pull request by simonl2002**
### Description

This change fixes the simple-processor example in the README. That example has a few issues that would cause the code to not compile and the processor from being successfully loaded by Conduit.

### Quick checks:

- [x] There is no other [pull request](https://github.com/conduitio/conduit-processor-sdk/pulls) for the same update/change.
- [ ] I have written unit tests. (n/a)
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,13 +112,13 @@ import (
 )
 
 func main() {
-   sdk.Run(&sdk.NewProcessorFunc(
-      sdk.Specification{Name: "simple-processor"}),
+   sdk.Run(sdk.NewProcessorFunc(
+      sdk.Specification{Name: "simple-processor", Version: "v1.0.0"},
```
Comment (hariso) [inline README.md:116]: @lovromazgon WDYT about `NewProcessorFunc` verifying that the spec contains a name and a version?

**Review by hariso — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (raulb) [reply README.md:116]: Maybe we could set default values if those are not provided? Or maybe just version. That way the processor would always pick the latest when not provided.

Comment (lovromazgon) [reply README.md:116]: Hmm, yeah maybe we can fill out the version in `NewProcessorFunc` in case it's not provided.

Comment (lovromazgon) [reply README.md:116]: https://github.com/ConduitIO/conduit-processor-sdk/pull/31

Comment (lovromazgon) [reply README.md:116]: Given that PR and given https://github.com/ConduitIO/conduit/pull/1439 we should be able to remove the version here @simonl2002.

Comment (simonl2002) [reply README.md]: Version removed

**Pull request merged**